### PR TITLE
Store config after creating a project.

### PR DESCRIPTION
**Pull request by cyrexin**
This PR adds the feature to store config per project. #129 

### Description of changes
1. Add a new field in the "nlpviewer_backend_project" table of the example database, which is used to store the config. 
image

2. When the user uploads the ontology, a default config is generated and the user can edit it.

3. When the user adds a new project, the config JSON will be stored in the database.

### Possible influences of this PR
n/a

### Demonstration of results
1. UI
store_config

2. Database
image

### CODE REVIEW

File: src/app/pages/Projects.tsx [hunk 1]
```diff
@@ -87,10 +89,34 @@ function Projects() {
...
     }
-  }  
+  }
+  
+  function createDefaultConfig(ontology: string) {
+    const ontologyJson = JSON.parse(ontology);
+    let config : IProjectConfigs = {legendConfigs: {}, scopeConfigs: {}};
+    for (const entry of ontologyJson.definitions) {
+      const entryName = entry.entry_name;
+      config['scopeConfigs'][entryName] = false;
```
Comment (hunterhector) [inline src/app/pages/Projects.tsx:104]: Only subtypes of "forte.data.annotation" can be used as scope, because these types contain "begin", "end".

**Review by hunterhector — COMMENTED** [review]
How will the config be used in the viewer?

File: src/app/pages/Projects.tsx (hunk 1 above)
Comment (cyrexin) [reply src/app/pages/Projects.tsx]: Done.

**Comment by cyrexin** [comment]
> How will the config be used in the viewer?

**Comment by hunterhector** [comment]
LGTM

**Review by hunterhector — APPROVED** [review]
LGTM

**Pull request merged**